### DISCUSSION TITLE
Error Lesson 6

### DISCUSSION THREAD

**Discussion by irwingtello** [smartcontractkit/full-blockchain-solidity-course-py#291] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/28800239/139589665-908dd9f3-a0ab-4aa2-bb4b-efda3b212581.png)

Hello , im facing with this error , i dont know how solve, i tried but my code looks good, can you help?

**Comment by cromewar** [comment, marked as answer]
It seems your problem is on your `get_account` function, which is returning accounts[1]... make sure it seems like this:

```Python
 import accounts

FORKED_LOCAL_ENVIRONMENTS = ["mainnet-fork", "mainnet-fork-dev"]
LOCAL_BLOCKCHAIN_ENVIRONMENTS = ["development", "ganache-local"]

def get_account():
    if (
        network.show_active() in LOCAL_BLOCKCHAIN_ENVIRONMENTS
        or network.show_active() in FORKED_LOCAL_ENVIRONMENTS
    ):
        return accounts[0]
    else:
        return accounts.add(config["wallets"]["from_key"])
```

also change ganache-local to ganache-gui,  and add it to LOCAL_BLOCKCHAIN_ENVIRONMENTS, this should work.
